src/htm/algorithms/Connections.hpp

```diff
@@ -428,12 +428,32 @@ class Connections : public Serializable
    * values until the desired number of synapses have permanences above the
    * connectedThreshold.  This is applied to a single segment.
    *
-   * @param segment  Index of segment on cell.   Is returned by method getSegment.
+   * @param segment  Index of segment in connections. Is returned by method getSegment.
    * @param segmentThreshold  Desired number of connected synapses.
    */
   void raisePermanencesToThreshold(const Segment    segment,
                                    const UInt       segmentThreshold);
 
+  /**
+   * Ensures that the number of connected synapses is sane.  This method
+   * controls the sparsity of the synaptic connections, which is important for
+   * the segment to detect things.  If there are too few connections then the
+   * segment will not detect anything, and if there are too many connections
+   * then the segment will detect everything.
+   *
+   * This method connects and disconnects synapses by uniformly changing the
+   * permanences of all synapses on the segment.  Synapses keep learning even
+   * after a limit is reached, at which point synapses change permanence
+   * relative to each other.
```

Comment:
> relative to each other. 

can you doc in the cpp where is this section? I don't see this part there?

Reply:
I'm removing that sentence because it's confusing and not really useful.